src/components/dashboard/Dashboard.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable indent */
```

Comment:
Generally, we don't want to disable eslint. If you want to reformat a file with VS Code, open the file and press `Shift + Alt + F`

Reply:
Hi Cole, all the files are properly indented but the indentation error keeps coming up. I have removed the line of code that disables eslint however